.github/workflows/backport.yml

```diff
@@ -11,6 +11,9 @@ on:
       - closed
 
 jobs:
+  get-branches:
+    uses: ./.github/workflows/branch-status-checks.yml
```

Comment:
It will run the whole workflow (aka 2 the jobs `list-target-branches` and `branch-status-checks`).

Currently it's not possible to [use a specific job](https://github.com/github-community/community/discussions/11072#discussioncomment-2127098) of a workflow. 

The alternative would be to just copy the contents from `list-target-branches`. 

I'd prefer to keep it as-is and wait until the option to select a specific job is available, but I can easily copy/paste the workflow also.